src/htsjdk/java/htsjdk/samtools/CRAMContainerStreamWriter2.java

```diff
@@ -442,11 +442,10 @@ protected void flushContainer() throws IllegalArgumentException {
             }
         }
 
-        final Container container = containerFactory.buildContainer(cramRecords);
-        for (final Slice slice : container.slices) {
+        final Container container = containerFactory.buildContainer(cramRecords, offset);
+        for (final Slice slice : container.getSlices()) {
```

Comment:
@tomwhite I think you can remove `CRAMContainerStreamWriter2` entirely, and use CRAMContainerStreamWriter` directly now that https://github.com/samtools/htsjdk/pull/1351 is merged ?

Reply:
Yes, of course! I'd forgotten about that change...